official/utils/misc/distribution_utils.py

```diff
@@ -120,6 +120,9 @@ def get_distribution_strategy(distribution_strategy="default",
           "flag cannot be set to 'off'.".format(num_gpus, num_workers))
     return None
 
+  if distribution_strategy == "noop":
```

Comment:
We already have "off", this  "noop" is not a path we are testing.  We could but I would not want to call it "no dist strat".  I am also winding down testing for the RC and just want to get coverage for the matrix we have so far.  This also seems like it would be really close to the "one execution" path; which we are also testing which will be the default.